pyproject.toml

```diff
@@ -12,6 +12,9 @@ authors = [
     "Ali R. Khan"
 ]
 readme = "README.md"
+packages = [
```

Comment:
Is this necessary? I would have expected poetry to be able to automatically pick it up.

Reply:
I tried to do it without this, but it didn't seem to pick it up in my tests.

Reply:
That's odd but if this works I'm fine with it